templates/terraform/property_documentation.erb

```diff
@@ -14,6 +14,15 @@
 <%   end -%>
 <% end -%>
 <%= indent(property.description.strip.gsub("\n\n", "\n"), 2) -%>
+<% if property.is_a?(Api::Type::Enum) && !property.output -%>
+<% unless property.default_value.nil? || property.default_value == "" -%>
```

Comment:
This ends up being a bit weird looking for some attributes where we want to describe the enum meanings (https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3404-old..auto-pr-3404?short_path=4ab0195#diff-4ab0195b34117c4f006f50cbfb766ae1), but I don't have a great solution there so I'm fine leaving it  as-is. Maybe look out for those and move those descriptions in line with the rest of the field description?